Presentation.Web/app/components/data-processing/tabs/data-processing-registration-sub-data-processor-modal.view.html

```diff
@@ -0,0 +1,49 @@
+﻿<div class="modal-header">
+    <div class="clearfix">
+        <h3 class="modal-title pull-left">{{::ctrl.title}}</h3>
+    </div>
+</div>
+
+<form name="createForm">
+    <div class="modal-body relation">
+        <div class="row">
+            <div class="col-md-3">
+
+            </div>
+            <!--Exposing System-->
+            <div class="col-md-6">
+                <div class="form-group">
+                    <div ng-if="!ctrl.isEdit">
+                        <label for="sub-data-processor_select-new">{{::ctrl.subDataProcessorFieldName}} *</label>
```

Comment:
since you can change sdp this will not work since it will be only rendered once

Reply:
actually it only changes when opening a new modal. I didn't want to write "Underdatabehandler" twice so I assigned it to a variable, but I now I'm not sure if it makes sense

Reply:
the error is with the cvr